src/containers/CounterButton.js

```diff
@@ -0,0 +1,25 @@
+import { connect } from 'react-redux';
+import {increaseCounter} from "../actions";
+import {decreaseCounter} from "../actions";
+import CounterButton from "../components/CounterButton";
+
+function mapDispatchToProps(dispatch){
```

Comment:
hopefully last class made this more clear. There should only be one mapDispatchToProps.